tests/e2e_test.go

```diff
@@ -223,7 +223,10 @@ func TestGet(t *testing.T) {
 	out := runCli(t, "upload", "--artifact", artifactPath, "--signature", sigPath, "--public-key", pubPath)
 	outputContains(t, out, "Created entry at")
 
-	uuid := getUUIDFromUploadOutput(t, out)
+	uuid, err := sharding.GetUUIDFromIDString(getUUIDFromUploadOutput(t, out))
```

Comment:
should `getUUIDFromUploadOutput` be renamed to `getEntryIDFromUploadOutput`?

Reply:
Agree, though this is a larger problem all throughout the code. #625 is tracking this. Pulling on that thread will definitely create enough changes for another PR.

Reply:
we can save it for later then!